src/components/document/editable-document-content.tsx

```diff
@@ -99,15 +99,14 @@ export const EditableDocumentContent: React.FC<IProps> = props => {
 
   const documentContext = useDocumentContext(document);
   const { db: { firebase }, ui, user } = useStores();
-  const { isNetworkedTeacher } = user;
 
   // set by the canvas and used by the toolbar
   const editableTileApiInterfaceRef: EditableTileApiInterfaceRef = useRef(null);
-
   const isReadOnly = !isPrimary || readOnly || document.isPublished;
   const isShowingToolbar = toolbar?.length && !isReadOnly;
   const showToolbarClass = isShowingToolbar ? "show-toolbar" : "hide-toolbar";
-  const isChatEnabled = isNetworkedTeacher;
+  const isChatEnabled = true;
```

Comment:
Don't we still want to restrict chat/comments to teachers?

Reply:
So, when Dennis uncovered the fact that it was easy to show to both teachers and students I went and asked (again) which the CMP team wanted. They have of course not answered me. Ship it either way and I'll force them to look at it and decide when next we meet.